ragna/assistants/_ollama.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import AsyncIterator, cast
+
+from ragna.core import RagnaException, Source
+
+from ._api import UnauthenticatedApiAssistant
+
+
+class OllamaApiAssistant(UnauthenticatedApiAssistant):
+    _MODEL: str
+
+    @classmethod
+    def display_name(cls) -> str:
+        return f"Ollama/{cls._MODEL}"
+
+    def _make_system_content(self, sources: list[Source]) -> str:
+        instruction = (
+            "You are an helpful assistants that answers user questions given the context below. "
+            "If you don't know the answer, just say so. Don't try to make up an answer. "
+            "Only use the following sources to generate the answer."
+        )
+        return instruction + "\n\n".join(source.content for source in sources)
+
+    async def _call_api(
```

Comment:
We don't need this extra layer of abstraction here. Just use `async def answer`. `ApiAssistant._call_api` is an artifact from a previous implementation that wasn't cleaned up. We can do that when we get to #381. But let's not replicate this mistake here.